src/net_packet.c

```diff
@@ -385,6 +382,11 @@ bool send_sptps_data(void *handle, uint8_t type, const void *data, size_t len) {
 			to->incompression = mesh->self->incompression;
 			return send_request(mesh, to->nexthop->connection, "%d %s %s %s -1 -1 -1 %d", ANS_KEY, mesh->self->name, to->name, buf, to->incompression);
 		} else {
+
+			if(to->mtu && (to->mtuprobes > 30)) {
```

Comment:
Why do you reset `to->mtuprobes` here?

Reply:
In one of our discussions we deduced that 2 instances or triggers where we can find sending via UDP failed are 
1. When we redirect UDP data via TCP
2. When ICMP sends message too big response for the UDP packet sent
Hence there we can rediscover the PMTU process and verify PMTU size.